docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3'
+
+services:
+  dev-node:
+    image: kiltprotocol/mashnet-node
```

Comment:
Can you add a commented out alternative, to use the develop branch on github?

Reply:
sure! I'd set it up so the image is built locally, so we don't shut people out who can't login to out AWS?

Reply:
I set this up in an override file which you can tell docker-compose to include with docker-compose -f dc.build.node.yml. For example docker-compose -f dc.build.node.yml up --build runs tests with a force rebuild of the mashnet-node from the current develop branch.
The built image is tagged local/mashnet-node, which we can hopefully re-use in other docker-compose files

Reply:
Looks good, can you please document it somewhere?

Reply:
done!